cam/concater.go

```diff
@@ -138,13 +138,13 @@ func (c *concater) concat(from, to time.Time, path string) error {
 
 	// Copy codec info from input to output context. This is necessary to ensure
 	// we do not decode and re-encode the video data.
-	for i := 0; i < int(inputCtx.nb_streams); i++ {
+	for i := range make([]int, int(inputCtx.nb_streams)) {
```

Comment:
I think this can be 
```suggestion
	for i := range int(inputCtx.nb_streams) {
```

Reply:
ah, thats a lot better -- will change range iterators to this pattern.